CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ Main (unreleased)
 - Fix bug where some capsule values (such as Prometheus receivers) could not
   properly be used as an argument to a module. (@rfratto)
 
+- Fix version information. (@rfratto)
```

Comment:
```suggestion
- Fix version information not displaying correctly when passing `--version` flag. (@rfratto)
```

Reply:
Thanks, I'll add this but it also impacts the `agent_build_info` metric, so I'll note that too 👍